src/mopidy/core/library.py

```diff
@@ -230,23 +230,23 @@ def lookup(self, uris: Iterable[Uri]) -> dict[Uri, list[Track]]:
         """
         validation.check_uris(uris)
 
-        futures = {}
+        futures = {
+            backend: backend.library.lookup(backend_uris)
```

Comment:
Doesn't this need to be the following to do anything? The other change to `wrapped_lookup` is just for typing.
```suggestion
            backend: backend.library.wrapped_lookup(backend_uris)
```
Assuming this is the case, I think it might need more tests.

Reply:
Probably a code smell in my current code, but the way I understand it was because a `BackendProxy` is passed here, `backend.library.lookup` here is the proxied method. In this case I proxied it to `wrapped_lookup`. The tests themselves are testing the expected inputs/outputs; but the actual logic of the try/catch in `wrapped_lookup` should be tested too I think. Maybe also a test that `wrapped_lookup` is ultimately called from the proxying? Maybe in `tests/backend/test_backend.py`? 

Alternatively just use `wrapped_lookup` instead and not try to proxy `lookup` to `wrapped_lookup`.

Reply:
Those class variables are for typing *only*. I may be mistaken but I don't see anything in the current tests that isn't either mocking out `lookup` or using `DummyBackend.lookup`, and the latter has been changed which now doesn't match the API (but makes test pass). You can see the Code Coverage report [here](https://app.codecov.io/github/mopidy/mopidy/commit/2d1f949eacde8d8cb1e35e74e251989810718d66) has no coverage of `wrapped_lookup` internals. 

In any case, when I ran Mopidy with these changes it threw an exception when core passed a list to `lookup()`.

It's worth adding support for the new API to the File Backend as part of this PR.

Reply:
Ah I understand now. Sorry this my first time working with pykka. Yea I see the issue,  I'll work on a change. And I'll update the file backend as well!